### PR TITLE
Document how we do a release

### LINKED ISSUE #209: Document how we do a "release"
Now that we are trying to do sprints and have clear "releases", we should document what steps need to be taken to perform the release.  What immediately comes to mind:

1. Update version number in `setup.py`
2. Update version number for ReadTheDocs
3. Create new tag/release on GitHub
4. Create new version of stable docs on ReadTheDocs
5. Write a changelog

### PR DESCRIPTION
**Pull request by bourque**
This PR adds documentation on how we define and perform a software release. 

The plan is to document this in a wiki page, but this PR also adds a new `CHANGES.rst` file that contains a changelog for the project.

Closes #209

### CODE REVIEW

**Comment by bourque** [comment]
The wiki page describing how to perform a release is here: https://github.com/spacetelescope/jwql/wiki/Software-Releases

**Comment by bourque** [comment]
@hover2pi Could you review this as well as the Wiki page I linked above?

**Comment by bourque** [comment]
@Johannes-Sahlmann @bhilbert4 @hover2pi Could one of you provide a review for this?  Pls thx

File: CHANGES.rst [hunk 1]
```diff
@@ -0,0 +1,245 @@
+0.16.0
+======
+
+This is the first release of the new release proceedures of ``jwql``.  The development team is now developing in release-driven sprints, so future releases will be less frequent, but with more changes
```
Comment (bhilbert4) [inline CHANGES.rst]: procEEdures

**Comment by bourque** [comment]
Thanks for the helpful comments @bhilbert4! 

RE: point 3:  Good catch.  I switched the wording to adhere to what the `conf.py` file suggests.

**Pull request merged**